tests/unittests/Foundation/NSExpressionTests.mm

```diff
@@ -14,132 +14,8 @@
 //
 //******************************************************************************
 
-#include <TestFramework.h>
+#import <TestFramework.h>
 #import <Foundation/Foundation.h>
-#import "ExpressionHelpers.h"
-
-void validateNSExpressionConstantValue(NSExpression* obj) {
```

Comment:
NSExpressionConstantValue is our own concoction.

---
In reply to: [72314636](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72314636) [](ancestors = 72314636)